README.md

```diff
@@ -1,13 +1,11 @@
 # Project Express API
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+First week with backend, building a api using Express.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+The API is organized using RESTful endpoints.
```

Comment:
Great that you did include RESTful as giving the viewer knowledge that you are aware of the meaning of RESTful API:s. Although it had been nice to have a little more description over how you worked with the project.